r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh

```diff
@@ -0,0 +1,49 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: requests-oauthlib
+# Version	: v1.3.0
+# Source repo	: https://github.com/requests/requests-oauthlib
+# Tested on	: UBI 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=requests-oauthlib
+PACKAGE_VERSION=${1:-v1.3.0}
+PACKAGE_URL=https://github.com/requests/requests-oauthlib
+
+yum -y update && yum install -y python3 python3-devel git gcc openssl-devel rust-toolset
+pip3 install tox
+
+HOME_DIR=`pwd`
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+		echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | Fail |  Clone_Fails"
+    	exit 0
+fi
+
+cd $HOME_DIR/$PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! tox -e py36; then
+	echo "------------------$PACKAGE_NAME:install_or_test_fails---------------------"
```

Comment:
Tox command is going to perform package testing only. Please refer file tox.ini for details.

For installation use following command, after line 37:
1. pip3 install ./

Please update the wordings, after adding installation command.